### PR TITLE
SASS-133 : Created Dividends submission audit event

### PR DESCRIPTION
**Pull request by robert-webb-hmrc**

### CODE REVIEW

File: app/controllers/dividends/DividendsCYAController.scala [hunk 1]
```diff
@@ -112,4 +119,11 @@ class DividendsCYAController @Inject()(
     }
   }
 
+  private def auditSubmission(details: CreateOrAmendDividendsAuditDetail)
+                                        (implicit hc: HeaderCarrier,
+                                         executionContext: ExecutionContext): Future[AuditResult] = {
+    val event = AuditModel("createOrAmendDividendUpdate", "createOrAmendDividendUpdate", details)
```
Comment (TheDeadlyPianist) [inline app/controllers/dividends/DividendsCYAController.scala:125]: Could you just add an 's' onto these so the spelling matches everything else?

```suggestion
    val event = AuditModel("createOrAmendDividendsUpdate", "createOrAmendDividendsUpdate", details)
```

File: conf/application.conf [hunk 2]
```diff
@@ -64,7 +64,7 @@ microservice {
       host = localhost
       port = 2003
       prefix = play.${appName}.
-      enabled = true
+      enabled = false
```
Comment (TheDeadlyPianist) [inline conf/application.conf:67]: I want this to stay false. Can you make a PR to set it to true in the base app conf please?

File: test/audit/AuditServiceSpec.scala [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+package audit
+
+import play.api.Configuration
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.audit.http.connector.AuditResult.Success
+import uk.gov.hmrc.play.audit.http.connector.{AuditConnector, AuditResult}
+import uk.gov.hmrc.play.audit.model.ExtendedDataEvent
+import utils.ViewTest
+import audit.AuditModel
```
Comment (TheDeadlyPianist) [inline test/audit/AuditServiceSpec.scala:26]: Unused import

**Review by TheDeadlyPianist — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 2 above)
Comment (robert-webb-hmrc) [reply conf/application.conf:67]: Looks to be the case already 👍
Screenshot 2020-11-27 at 13 45 19

**Pull request merged**